packages/utils/src/shorthand.ts

```diff
@@ -0,0 +1,616 @@
+    'overscroll-behavior-inline',
+    'overscroll-behavior-block',
+  ],
+  padding: [
+    'padding-block',
+    'padding-block-end',
+    'padding-block-start',
+    'padding-bottom',
+    'padding-inline',
+    'padding-inline-end',
+    'padding-inline-start',
+    'padding-left',
+    'padding-right',
+    'padding-top',
+  ],
+  'padding-block': ['padding-block-start', 'padding-block-end', 'padding-top', 'padding-bottom'],
+  'padding-inline': ['padding-inline-start', 'padding-inline-end', 'padding-left', 'padding-right'],
+  'place-content': ['align-content', 'justify-content'],
+  'place-items': ['align-items', 'justify-items'],
+  'place-self': ['align-self', 'justify-self'],
+  'position-try': ['position-try-order', 'position-try-fallbacks'],
+  'scroll-margin': [
+    'scroll-margin-block',
+    'scroll-margin-block-end',
+    'scroll-margin-block-start',
+    'scroll-margin-bottom',
+    'scroll-margin-inline',
+    'scroll-margin-inline-end',
+    'scroll-margin-inline-start',
+    'scroll-margin-left',
+    'scroll-margin-right',
+    'scroll-margin-top',
+  ],
+  'scroll-margin-block': [
+    'scroll-margin-block-start',
+    'scroll-margin-block-end',
+    'scroll-margin-top',
+    'scroll-margin-bottom',
+  ],
+  'scroll-margin-inline': [
+    'scroll-margin-inline-start',
+    'scroll-margin-inline-end',
+    'scroll-margin-left',
+    'scroll-margin-right',
+  ],
+  'scroll-padding': [
+    'scroll-padding-block',
+    'scroll-padding-block-end',
+    'scroll-padding-block-start',
+    'scroll-padding-bottom',
+    'scroll-padding-inline',
+    'scroll-padding-inline-end',
+    'scroll-padding-inline-start',
+    'scroll-padding-left',
+    'scroll-padding-right',
+    'scroll-padding-top',
+  ],
+  'scroll-padding-block': [
+    'scroll-padding-block-start',
+    'scroll-padding-block-end',
+    'scroll-padding-top',
+    'scroll-padding-bottom',
+  ],
+  'scroll-padding-inline': [
+    'scroll-padding-inline-start',
+    'scroll-padding-inline-end',
+    'scroll-padding-left',
+    'scroll-padding-right',
+  ],
+  'scroll-timeline': ['scroll-timeline-name', 'scroll-timeline-axis'],
+  'text-decoration': [
+    'text-decoration-color',
+    'text-decoration-line',
+    'text-decoration-style',
+    'text-decoration-thickness',
+  ],
+  'text-emphasis': ['text-emphasis-color', 'text-emphasis-style'],
+  'text-wrap': ['text-wrap-mode', 'text-wrap-style'],
+  transition: [
+    'transition-behavior',
+    'transition-delay',
+    'transition-duration',
+    'transition-property',
+    'transition-timing-function',
+  ],
+  'view-timeline': ['view-timeline-name', 'view-timeline-axis'],
+};
+
+// Make sure to update the `shorthandBuckets` variable in `packages/react/src/runtime/shorthand.ts`
+// after making changes to this object.
+//
+// You may want to use `packages/scripts/src/generate-shorthand.ts` to help
+// you generate this object.
+export const shorthandBuckets: Record<ShorthandProperties, Depths> = {
```

Comment:
Could you import just the type of this to make sure they're 1:1?

Reply:
it doesn't look like it would achieve anything unfortunately... 

the two copies of `shorthandBuckets` are already type annotated as `Record<ShorthandProperties, Depths>`, so the keys are guaranteed to be the same (but the values aren't)

if i were to remove the type annotation from one of the `shorthandBuckets`, and set the type of the other `shorthandBuckets` to `typeof importedShorthandBuckets` (where `importedShorthandBuckets` points to the first `shorthandBuckets`), the typing would be achieve almost the same result as if we just annotated the two variables as `Record<ShorthandProperties, Depths>`
![Screenshot 2024-09-26 at 10 56 08](https://github.com/user-attachments/assets/c6426c20-390c-44f9-942e-1389249e1a78)

maybe an automated PR check could enforce that they be 1:1

Reply:
This might be a bit of magic, but it should work:
```tsx
export const shorthandBuckets = {
  all: 0,
} as const satisfies Record<ShorthandProperties, Depths>;
```

```tsx
import type {
  Depths,
  ShorthandProperties,
  shorthandBuckets as ShorthandBucketsType,
} from '@compiled/utils';

const shorthandBuckets: typeof ShorthandBucketsType = {
```

<img width="668" alt="Screenshot 2024-09-26 at 1 11 02 PM" src="https://github.com/user-attachments/assets/38d43033-df1e-46a3-b352-1e2b6053ecc5">

<img width="664" alt="Screenshot 2024-09-26 at 1 10 58 PM" src="https://github.com/user-attachments/assets/99a70e02-b72b-45fb-a43f-2e190dab1a43">

Reply:
you're a genius!! 😄